timesketch/lib/analyzers/contrib/hashlookup_analyzer_test.py

```diff
@@ -0,0 +1,69 @@
+"""Test for Hashlookup"""
+
+import copy
+import mock
+
+from flask import current_app
+
+from timesketch.lib.analyzers.contrib import hashlookup_analyzer
+from timesketch.lib.testlib import BaseTest
+from timesketch.lib.testlib import MockDataStore
+
+SHA256_HASH = "ac7233de5daa4ab262e2e751028f56a7e9d5b9e724624c1d55e8b070d8c3cd09"
+SHA256_N_HASH = "bc7233de5daa4ab262e2e751028f56a7e9d5b9e724624c1d55e8b070d8c3cd09"
+MATCHING_HASH = {"sha256_hash": SHA256_HASH}
+NO_MATCHING_HASH = {"sha256_hash": SHA256_N_HASH}
+
+
+class TestHashlookup(BaseTest):
+    """Tests the functionality of the analyzer."""
+
+    def setUp(self):
+        super().setUp()
+        current_app.config["HASHLOOKUP_URL"] = "blah"
+
+    @mock.patch("timesketch.lib.analyzers.interface.OpenSearchDataStore", MockDataStore)
+    @mock.patch(
+        "timesketch.lib.analyzers.contrib.hashlookup_analyzer."
+        "HashlookupAnalyzer.get_hash_info"
+    )
```

Comment:
mocking `get_hash_info` is probably not the best decision here, given that it's really part of what you wanna test. In fact, given that it's a pretty atomic function it's easy to test! 

I would suggest mocking `requests.get` instead, and

* making sure that the mock is called with the expected parameters (e.g. the URL)
* Asserting that the return value of `get_hash_info` is as expected, depending on the `return_value` of the `requests.get` mock

Reply:
Fixed in 2927ba1189072618de1d0de9d0a4b59d257b1d78.